### DISCUSSION TITLE
is this platform works as same functionallity woocomerce does?

### DISCUSSION THREAD

**Discussion by loveyandex** [vendurehq/vendure#850] [Q&A]
we want use your solution instead of wp woocomerce. is it complete project now . i see ot is 99% ok.
do you think what is lack in this project

**Comment by michaelbromley** [comment]
The main difference between this an WooCommerce is that Vendure does not handle the storefront part at all. So you are responsible for building a storefront app.

Another difference is that there is no marketplace for extensions like with WooCommerce. Our approach is a little different - we are more focused on creating an excellend developer experience and making it simpler to build custom functionality yourself rather than having to rely on an extensions marketplace.

In terms of functionality I'm not totally familiar with what WC offers.

**Reply by loveyandex** [reply]
do you support multi sellers for a product?

**Reply by michaelbromley** [reply]
We have a "channels" feature which allows you to assign a product to multiple channels, and each channel can represent a seller.

**Reply by lusicsmusics1** [reply]
is in real world channels uses for multi seller?

**Reply by michaelbromley** [reply]
Yes, there are a few projects I have heard of in development using channels for this purpose.

**Reply by loveyandex** [reply]
> We have a "channels" feature which allows you to assign a product to multiple channels, and each channel can represent a seller.

so how can we define seller as a user
seller1 seller 2 ...?

**Reply by michaelbromley** [reply]
You can create an Administrator for each seller, and create a Role which is limited to their Channel only. This allows them to create products etc which are available in their Channel but will not be visible to other sellers.
